mmv1/templates/terraform/examples/image_basic_storage_location.tf.erb

```diff
@@ -0,0 +1,8 @@
+resource "google_compute_image" "example" {
+  name = "<%= ctx[:vars]['image_name'] %>"
+
+  raw_disk {
+    source = "https://storage.googleapis.com/bosh-gce-raw-stemcells/bosh-stemcell-97.98-google-kvm-ubuntu-xenial-go_agent-raw-1557960142.tar.gz"
```

Comment:
This test seems to rely on an external disk image in a personal project. Is that so?

Reply:
Ahh nevermind.. This value is already used in other tests, apologies